views/chorals.pug

```diff
@@ -0,0 +1,34 @@
+extends layout
+
+block content
+  // !PAGE CONTENT!
+  header.w3-container(style='padding-top:22px')
+    h5
+      b
+        i.fa.fa-dashboard
+        |  My Chorals
+  hr
+  .w3-container
+    h5 Chorals
+    table.w3-table.w3-striped.w3-bordered.w3-border.w3-hoverable.w3-white
+      tr
+        td Name
+        td Type
+        td ID
+        td Sample Rate
+        td Function
+      each choral in chorals
+        tr
+          td= choral.name
+          td= choral.choralType
+          td= viewHelpers.truncateWithDots(choral.choralId, 30)
+          td= choral.sampleRate
+          td= choral.func
+  // Footer
+  footer.w3-container.w3-padding-16.w3-light-grey
+    h4 Powered by Choral Cloud
+    p
+      a.w3-btn.w3-black(href='https://github.com/ChoralCloud', target='_blank')
+        i.fa.fa-code-fork.w3-large
+        |  Check out the code
```

Comment:
Sry didn't see this. This is just boilerplate. It's the page footer. Should move this to the layout actually.